index.js

```diff
@@ -26,32 +26,36 @@ module.exports = {
     }
   },
 
-  setupPreprocessorRegistry(type, registry) {
-    if (type === 'parent') {
-      this._assignOptions(registry.app);
-
-      if (this._stripTestSelectors) {
-        let StripTestSelectorsTransform = require('./strip-test-selectors');
-
-        registry.add('htmlbars-ast-plugin', {
-          name: 'strip-test-selectors',
-          plugin: StripTestSelectorsTransform,
-          baseDir() { return __dirname; }
-        });
-      } else {
-        let TransformTestSelectorParamsToHashPairs = require('./transform-test-selector-params-to-hash-pairs');
+  _setupPreprocessorRegistry(registry) {
+    if (this._stripTestSelectors) {
+      let StripTestSelectorsTransform = require('./strip-test-selectors');
+
+      registry.add('htmlbars-ast-plugin', {
```

Comment:
Seems like this could be made a bit simpler/easy to read:

```js
let transform = 'transform-test-selector-params-to-hash-pairs';

if (this._stripTestSelectors) {
  transform = 'strip-test-selectors';
}

registry.add('htmlbars-ast-plugin', {
  name: transform,
  plugin: require(`./${transform}`),
  baseDir() { return __dirname; }
});
```